docs/source/changelog.rst

```diff
@@ -1,427 +1,138 @@
-#########
```

Comment:
I think we should keep the old changelog somewhere

Reply:
Indeed. We can maybe think about splitting the changelog into different files for different versions, but I am also against simply overriding the current file.

Reply:
This was my bad. I accidentally added these changes in 7d9d605. Feel free to revert this file for now.

I think changelog.rst should just be a `.. toctree::`  linking to the version of the generated changelog (e.g. 0.5.0-changelog.rst) and the older version should be renamed and/or split into multiple files? What do you guys think?

Reply:
I think that the script, writing it to a file, I think it should be better than we print it on screen and ask the release managers to place it accordingly.
What I think is to create a folder called changelog, have this as `changelog-0.1.0-0.4.0.rst` and newer changelogs will have `release-notes-0.5.0.rst`. Thoughts?

Reply:
For the sake of consistency, I'd rather split all of the previous section into separate files. But otherwise I am all for it. 👍

Reply:
I reverted the changelog (95b02a4) in case we'd like to revisit it, but I've also gone ahead and done two small things ():

1. Split the previous section into separate files
2. Made the script output `{cur_release[1:]}-changelog.rst` by default (remove the `v` from the git tag/revision and place it in the appropriate directory under a `:reversed:` `:glob:`... might want to set a depth.
![image](https://user-images.githubusercontent.com/29168469/112850067-86556100-905e-11eb-84da-0e90d83addaa.png)

Reply:
> remove the v from the git tag/revision

What do you mean by this?

Reply:
https://github.com/ManimCommunity/manim/pull/1138/commits/be5baa70183c5acf86bddc497c029ee19d7227eb#diff-c461d965355498ee3bd152eda8723432a8d81b2bf09246f0cc8bf38c96d406a4R162